chart/kgb/templates/operator.yaml

```diff
@@ -36,21 +36,21 @@ spec:
                 fieldRef:
                   fieldPath: metadata.name
             - name: OPERATOR_NAME
-              value: "ohmyglb"
-            - name: OHMYGLB_VERSION
+              value: "kgb"
+            - name: kgb_VERSION
```

Comment:
Suggestion: `KGB_VERSION`, as we use allcaps here.

Reply:
@somaritane yeah, part of the caps are broken due to automated conversion, will tweak the commands